web/content/index.adoc

```diff
@@ -20,21 +20,18 @@ For contribution, visit the https://github.com/theforeman/foreman-documentation[
 
 The following releases are supported, meaning they receive (security) updates.
 
+* link:/release/3.10/[Foreman 3.10 & Katello 4.12]
 * link:/release/3.9/[Foreman 3.9 & Katello 4.11]
-* link:/release/3.8/[Foreman 3.8 & Katello 4.10]
 
 === Unsupported releases
 
 The future version is built in nightly.
 
 * link:/release/nightly/[Nightly]
 
-There is a release candidate available for testing.
-
-* link:/release/3.10/[Foreman 3.10 & Katello 4.12]
-
 These releases are unsupported and no longer receive updates. Users should update to a supported release.
```

Comment:
```suggestion
// There is a release candidate available for testing.

These releases are unsupported and no longer receive updates. Users should update to a supported release.
```

I cannot comment on deleted lines; but it'd be nice to keep line 32 as a comment that we can reuse for the next RC.

Reply:
Considered this. I'll add it back